event/event.go

```diff
@@ -152,8 +154,13 @@ func NewEvent(ctx context.Context, ef store.EventFile, hub lab.Hub) (Event, erro
 		closers:       []io.Closer{ctf, guac, hub, keyLoggerPool},
 		dockerHost:    dockerHost,
 		keyLoggerPool: keyLoggerPool,
+		teamQueue:     make(chan *store.Team, conf.Capacity),
 	}
 
+	hub.AttachHook(func() error {
+		return ev.processQueue()
```

Comment:
This may have unwanted effects, as this error has nothing to do with adding and removing vm's.